README.md

```diff
@@ -1,66 +1,55 @@
-# IdP Link
+# WorkOS.js Example
+## Overview
 
-WorkOS offers a Javascript based embeddable widget that allows your Enterprise users to configure their Single Sign-On Identity Provider themselves. Using IdP Link allows you to reduce the back and forth between your team and your new Enterprise customer, getting the Enterprise onboarded quicker and using your application.
+WorkOS.js allows your end-users to set up and configure SSO connections, all without leaving your application.
 
-![alt text](./IdPLink.gif)
-
-
-### Configuring Identity Providers for WorkOS SSO
-
-Whenever you need to onboard a new Enterprise, you'll need to exchange some information with the Enterprise to set up their SSO Identity Provider. You'll create a secure handshake between your application and the Enterprise's Identity Provider via WorkOS.
+This creates a seamless "self-serve" process for upgrading users to SSO, and removes the complex back-and-forth communication otherwise required. It's the fastest and easiest way to get your enterprise users signing in with SSO.
 
-First, WorkOS will generate and you'll provide an Enterprise with an Assertion Consumer Service (ACS) URL. This tells the IdP where to redirect the Enterprise user with an authentication response when they wish to sign in to your app. WorkOS provides this value to you, and you must provide it to the Enterprise.
-
-Next, the Enterprise must create and configure your application in their Identity Provider. They'll use the ACS URL you provide, as well as some other pieces of information specific to the provider that WorkOS will also provide as needed.
-
-Once complete, the Enterprise will have all of the information needed to finish configuring the IdP via WorkOS. The Enterprise can relay this information to your team, who can then use that information to complete the IdP configuration on the WorkOS dashboard.
-
-Instead, this repository demonstrates how to use WorkOS IdP Link to allow the Enterprise to enter this information themselves.
+![alt text](./IdPLink.gif)
 
-### Installation
+To use WorkOS.js, you'll need to know your `Project ID`, your `Publishable Key`. You'll also need to supply the specific domain of an enterprise customer who is creating the new Identity Provider connection. Let's get started.
 
-WorkOS IdP Link is available as a Javascript embed via the WorkOS CDN. WorkOS is always adding new Identity Providers and keeping the embed updated, so it's important you include the script from the WorkOS CDN as opposed to copying it into your codebase. Include the following script tag on your website:
+## Set up the Element
 
-```html
-<script async type="text/javascript" src="https://cdn.workos.com/idp-link.min.js"</script>
+Add the script to your application's website.
```

Comment:
can we add something like "and include on any page where you allow your user to set up and configure SSO connections." Then something about async or defer, and where to put on page etc. This is a nice article you can borrow from - https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/loading-third-party-javascript

Reply:
Sure